src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -182,24 +198,69 @@ const TableToolbar = ({
         disabled={isDisabled}
       />
     ) : null}
-    <StyledTableToolbarContent>
-      {customToolbarContent || null}
-      {totalFilters > 0 ? (
-        <Button kind="secondary" onClick={onClearAllFilters}>
-          {i18n.clearAllFilters}
-        </Button>
-      ) : null}
-      {hasColumnSelection ? (
-        <ToolbarSVGWrapper onClick={onToggleColumnSelection}>
-          <IconColumnSelector description={i18n.columnSelectionButtonAria} />
-        </ToolbarSVGWrapper>
-      ) : null}
-      {hasFilter ? (
-        <ToolbarSVGWrapper onClick={onToggleFilter}>
-          <IconFilter description={i18n.filterButtonAria} />
-        </ToolbarSVGWrapper>
-      ) : null}
-    </StyledTableToolbarContent>
+    {activeBar === 'edit' ? (
+      <StyledTableToolbarContent>
+        <StyledCarbonButton kind="ghost" onClick={onCancelEditAction}>
+          {i18n.batchCancel}
+        </StyledCarbonButton>
+        <StyledCarbonButton onClick={onSaveEditAction}>{i18n.batchSave}</StyledCarbonButton>
+      </StyledTableToolbarContent>
+    ) : activeBar === 'undo' ? (
+      <StyledCarbonToastNotification
+        caption=""
+        hideCloseButton={false}
+        iconDescription="undo changes and close"
+        kind="success"
+        lowContrast
+        notificationType="toast"
+        onCloseButtonClick={onUndoEditAction}
+        role="alert"
+        style={{
+          marginBottom: '.5rem',
```

Comment:
These styles should be extracted into a Sass partial for the component. [link to contributing.md](https://github.com/IBM/carbon-addons-iot-react/blob/master/.github/CONTRIBUTING.MD#styling-new-components) 
Values should be carbon design token variables instead of hardcoded values.

Reply:
Yes, I've extracted the styles into `_table.scss`. BTW, what is carbon design token variables? Am I right now?